plugins/base/src/test/kotlin/expect/ExpectUtils.kt

```diff
@@ -0,0 +1,55 @@
+package expect
+
+import org.junit.jupiter.api.Assertions.assertTrue
+import testApi.testRunner.AbstractCoreTest
+import java.nio.file.Files
+import java.nio.file.Path
+import java.nio.file.Paths
+import java.util.concurrent.TimeUnit
+import kotlin.streams.toList
+
+open class ExpectUtils : AbstractCoreTest() {
```

Comment:
1) This could probably be an `abstract class` which is IMHO better in this context as creating an instance of this class has no visible use cases for me
2) It should not be named `ExpectUtils` as it isn't a utility class. Either make proper utils from this or name it something like `AbstractExpectTest`

Reply:
This class should be splitted into base class for this kind of tests and top level utility functions in separate file.

Reply:
I've rewritten it, now it's AbstractExpectTest and ExpectUtils.kt with top level functions.